nexus/tests/integration_tests/console_api.rs

```diff
@@ -177,39 +198,20 @@ async fn test_console_pages(cptestctx: &ControlPlaneTestContext) {
     ];
 
     for path in console_paths {
-        let console_page = RequestBuilder::new(&testctx, Method::GET, path)
-            .header(http::header::COOKIE, session_token.clone())
-            .expect_status(Some(StatusCode::OK))
-            .expect_response_header(
-                http::header::CONTENT_TYPE,
-                "text/html; charset=UTF-8",
-            )
-            .expect_response_header(http::header::CACHE_CONTROL, "no-store")
-            .execute()
-            .await
-            .expect("failed to get console index");
-
-        assert_eq!(console_page.body, "<html></html>".as_bytes());
+        expect_console_page(testctx, path, Some(session_token.clone())).await;
     }
 }
 
 #[nexus_test]
-async fn test_login_form(cptestctx: &ControlPlaneTestContext) {
+async fn test_unauthed_console_pages(cptestctx: &ControlPlaneTestContext) {
     let testctx = &cptestctx.external_client;
 
-    // login route returns bundle too, but is not auth gated
-    let console_page =
-        RequestBuilder::new(&testctx, Method::GET, "/spoof_login")
-            .expect_status(Some(StatusCode::OK))
-            .expect_response_header(
-                http::header::CONTENT_TYPE,
-                "text/html; charset=UTF-8",
-            )
-            .execute()
-            .await
-            .expect("failed to get login form");
+    let unauthed_console_paths =
+        &["/spoof_login", "/login/irrelevant-silo/local"];
```

Comment:
The changes in this file are mostly cleanup. This line is the only substantive change: check the new path in addition to `/spoof_login`.